### PR TITLE
rpc: Assert named arguments are unique in RPCHelpMan

### PR DESCRIPTION
**Pull request by promag**
Prevents an obvious mistake.

### CODE REVIEW

File: src/rpc/util.cpp [hunk 1]
```diff
@@ -249,7 +249,9 @@ std::string RPCHelpMan::ToString() const
     // Oneline summary
     ret += m_name;
     bool is_optional{false};
+    std::set<std::string> args;
     for (const auto& arg : m_args) {
+        assert(args.insert(arg.m_name).second);
```
[the remark this answers is not in the document]
Comment (promag) [reply src/rpc/util.cpp]: Sure, will do.

**Comment by Empact** [comment]
nit: might be a bit better to have this validation in the constructor, from a separation of concerns perspective.

**Comment by Empact** [comment]
utACK e09a5875cac3e691bb9cc8301cb882d63b029cb0

**Comment by MarcoFalke** [comment]
utACK e09a587

**Pull request merged**